### PR TITLE
doc: add ytt overlay example for TLS certs

### PR DESCRIPTION
**Pull request by kauana**
Hi friends,

@tcdowney and I want your opinion on how this doc looks. More specifically, we would like to know how we can better document how operators should apply ytt overlays.

cc @rodolfo2488 @christianang @XanderStrike @ndhanushkodi @rosenhouse

### CODE REVIEW

File: doc/managing-certificates-for-application-ingress.md [hunk 1]
```diff
@@ -12,21 +12,35 @@ kubectl create secret tls wildcard-apps-example-com-cert \
...
+    port:
+      name: https-apps-example-com
+      number: 443
+      protocol: HTTPS
+    tls:
+      credentialName: wildcard-apps-example-com-cert
+      mode: SIMPLE
 ```
 
+This overlay can be applied during installation of
```
Comment (kauana) [inline doc/managing-certificates-for-application-ingress.md:41]: @jenspinney @syerram we would like to know how you would expect operators to apply overlays?

Comment (XanderStrike) [reply doc/managing-certificates-for-application-ingress.md:41]: wish: a link to a generic process doc for "applying ytt overlays to cf-for-k8s deployments"

Comment (Syerram) [reply doc/managing-certificates-for-application-ingress.md:41]: Currently, the install script does not accept overlays parameters. 

FWIW, this looks ok to me.

In the long term, we will move away from the current install script and expect end-users to run `ytt | kapp` directly. We will provide documentation for operators on how they can add overlays to ytt (via -f)  among other things.

Comment (loewenstein) [reply doc/managing-certificates-for-application-ingress.md:41]: Technically, the parameter that the script takes is passed to ytt as -f which can be either a file or a directory.

That's how we currently provide overlays and values to the script.

**Pull request merged**